includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\PHP_CodeSniffer_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+use WordPress\Plugin_Check\Checker\Check;
+use WordPress\Plugin_Check\Checker\Check_Result;
+use Exception;
+
+/**
+ * Check for running one or more PHP CodeSniffer sniffs.
+ *
+ * @since n.e.x.t
+ */
+abstract class Abstract_PHP_CodeSniffer_Check implements Check {
+
+	/**
+	 * List of allowed PHPCS arguments.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @var array
+	 */
+	protected $allowed_args = array(
+		'extensions',
+		'standard',
+		'sniffs',
+		'exclude',
+	);
+
+	/**
+	 * Returns an associative array of arguements to pass to PHPCS.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
```

Comment:
Please include a brief description on what that array should include. Specifically since this is an abstract method, it should include details on the format of the data.

See for example https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/post.php#L1552 on how WP core formats associative array doc-blocks, we could use that here too (except that here we do it for a `@return` doc instead of `@param`.